src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -330,13 +330,13 @@ class Filter extends Component<Props> {
                               )
                             ) : (
                               // Desktop
-                              <Sidebar width="30%" mr={2}>
+                              <Sidebar width="25%" mr={2}>
                                 <Filters />
                               </Sidebar>
                             )}
 
                             {/* Main Artwork Grid */}
-                            <Box width={xs ? "100%" : "70%"}>
+                            <Box width={xs ? "100%" : "75%"}>
```

Comment:
Why are these changes being made here? To me they don’t really seem a part of what this PR is about.

Reply:
This is a little thing Brian and I were chatting about in Slack but your right, will remove 👍